webcompat/static/js/lib/autogrow-textfield.js

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*eslint new-cap: ["error", { "capIsNewExceptions": ["Deferred"] }]*/
```

Comment:
We don't need this exception for `Deferred` in this file, right?

Reply:
(not a big deal, if you want to push a fixup commit then merge, go for it. ^__^)